test/test_models.py

```diff
@@ -0,0 +1,76 @@
+import unittest
+
+import numpy as np
+import pandas as pd
+from pandas.testing import assert_frame_equal, assert_series_equal
+from sklearn.datasets import load_breast_cancer
+
+from pyro_risks.models import score_v0
+
+
+class UtilsTester(unittest.TestCase):
+    def test_prepare_dataset(self):
+        df = pd.DataFrame(
+            {'day': ['2019-07-01', '2019-08-02', '2019-06-12'],
+             'departement': ['Aisne', 'Cantal', 'Savoie'],
+             'fires': [0, 5, 10],
+             'fwi_mean': [13.3, 0.9, 2.5],
+             'ffmc_max': [23, 45.3, 109.]
+             })
+        X, y = score_v0.prepare_dataset(df, selected_dep=['Aisne', 'Cantal'])
+        assert_frame_equal(
+            X, pd.DataFrame({'ffmc_max': {0: 23.0, 1: 45.3}, 'fwi_mean': {0: 13.3, 1: 0.9}}))
+        assert_series_equal(y, pd.Series([0, 1], name='classif_target'))
```

Comment:
perhaps there is a ̀frame_equal` boolean function in pandas we could use? (To keep the `self.assertTrue`)
Same for following tests

Reply:
what is your point? `assert_frame_equal`and `assert_series_equal` are pandas testing functions which do the job...

Reply:
that the method could be a function as is, but it's really just a detail :ok_hand: